spec/views/hq/sponsors/index_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'will_paginate/array'
```

Comment:
Sorry, that must be `build_stubbed_list` that's doing that.  I'd leave this here then.